core/common/src/main/java/alluxio/conf/AlluxioConfiguration.java

```diff
@@ -35,12 +35,15 @@
   String get(PropertyKey key);
 
   /**
-   * Gets the value for the given key in the {@link Properties}; if this key is not found, a
-   * RuntimeException is thrown.
-   *
-   * @param key the key to get the value for
-   * @param options options for getting configuration value
-   * @return the value for the given key
+   * Gets the value for the given key in the properties of this configuration.
+   * <p>
+   * Implementations should return the value mapped to the provided {@code key}
+   * with the desired configuration value {@code options}.
+   *
+   * @param key the {@link PropertyKey} to get the value from
+   * @param options the {@link ConfigurationValueOptions} for getting the
+   *                configuration value from the {@code key}
```

Comment:
The second line of text should align with the first argument after the tag.

References: [https://github.com/Alluxio/alluxio/pull/11797/files](https://github.com/Alluxio/alluxio/pull/11797/files)